doc-content/enterprise-only/getting-started/assets-types-ref.adoc

```diff
@@ -99,7 +95,7 @@ Test scenarios in {PRODUCT} enable you to validate the functionality of rules, m
 
 * Test Scenario (Legacy)
 +
-Starting in {PRODUCT} {PRODUCT_VERSION}, there is a new default Test Scenario asset. As it's still not fully feature complete, {PRODUCT} {PRODUCT_VERSION} also includes support for the Test Scenario. Thus providing two test scenarios for business processes, the _Test Scenario_, which relies on the new default process designer, and _Test Scenario (legacy)_, which relies on the legacy process designer.
+Because the new default Test Scenario asset is still not fully feature complete, {PRODUCT} {PRODUCT_VERSION} includes support for the legacy Test Scenario.
```

Comment:
Minor suggestion - reverse the sentence order and rephraser:

s/Because the new default Test Scenario asset is still not fully feature complete, {PRODUCT} {PRODUCT_VERSION} includes support for the legacy Test Scenario./{PRODUCT} {PRODUCT_VERSION} includes support for the legacy Test Scenario because the default Test Scenario asset is still in development.

Reply:
Changed per suggestion. Thank you.